src/macosx/osxgl.m

```diff
@@ -2318,7 +2318,9 @@ static void set_icons(ALLEGRO_DISPLAY *display, int num_icons, ALLEGRO_BITMAP* b
    NSAutoreleasePool *pool = [[NSAutoreleasePool alloc] init];
    NSImage *image = NSImageFromAllegroBitmap(bitmap);
    (void)display;
-   [NSApp setApplicationIconImage: image];
+   [NSApp performSelectorOnMainThread: @selector(setApplicationIconImage:)
+                           withObject: image
```

Comment:
Do you mind fixing the indentation here and in the other file?

Reply:
Not sure if this was what you wanted but I have indented 3 spaces instead of what was there before (Xcode likes to auto-align the colons, I left it because system.m was done that way)
https://github.com/liballeg/allegro5/blob/7e3ba4b5856bf8ac1008bed5447bb26477a8a5af/src/macosx/system.m#L66-L72